source/slides/03_linux_basics.rst

```diff
@@ -0,0 +1,443 @@
+.. _03_linux_basics:
+
+Linux Basics (Day 3)
+====================
+
+Filesystems, system booting and OS Installation
+
+Before we begin
+---------------
+
+.. code-block:: bash
+
+  $ sudo yum install man man-pages man-pages-overrides
```

Comment:
This doesn't make as much sense, since they won't have their laptops right in front of them. It was mainly for having DOBCers install man so they could follow along on some man pages.